resources/servo.css

```diff
@@ -132,7 +132,7 @@ textarea {
   -servo-overflow-clip-box: content-box;
 }
 
-// https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements
```

Comment:
No problem, if I had to remember the comment syntax in every language I use, I don't think my brain would have space for much else!